src/Functions/generateSerialID.cpp

```diff
@@ -164,14 +179,15 @@ REGISTER_FUNCTION(Serial)
     {
         .description=R"(
 Generates and returns sequential numbers starting from the previous counter value.
-This function takes a constant string argument - a series identifier.
+This function takes a constant string argument - a series identifier, and an optional starting value.
```

Comment:
Let's mention that the start value is ignored if the counter already exist, similarly to how you added the example.

Reply:
Yeah :+1: I would add that to the argument description.